chainer/iterators/multiprocess_iterator.py

```diff
@@ -94,7 +94,10 @@ def __next__(self):
             self._previous_epoch_detail, self._order) = prefetch_state
         if batch is None:
             raise StopIteration
-        else:
+
+        def dummy_state():
+            raise NotImplementedError
+        with random_state.set_random_state(dummy_state):
```

Comment:
It seems to me this context manager doesn't take any effect on the return `batch`. If so, are you thinking to modify the code in the PRs following this one?